apps/controllerx/devices/ikea.py

```diff
@@ -13,6 +13,7 @@ class E1810Controller(LightController):
     def get_z2m_actions_mapping(self):
         return {
             "toggle": Light.TOGGLE,
+            "toggle_hold": Light.ON_FULL_BRIGHTNESS,
```

Comment:
When holding the button. It sends `hold` straight away and after 3 seconds it sends `toogle_hold`. So this means that if the light is on, you will experience that ehe light will turn off when holding the button and after 3 seconds it will turn the light again to full brightness. For this reason, I did not add this behaviour to the default controller, it is rather confusing.
```suggestion
```